bridgesample/src/main/java/com/livefront/bridgesample/scenario/fragment/NonBridgeLargeDataFragment.kt

```diff
@@ -26,20 +29,50 @@ class NonBridgeLargeDataFragment : NonBridgeBaseFragment() {
             setHeaderText(R.string.non_bridge_large_data_header)
             generatedBitmap = savedBitmap
             onBitmapGeneratedListener = { savedBitmap = it }
+            if (getArguments(this@NonBridgeLargeDataFragment).infiniteBackstack) {
+                onNavigateButtonClickListener = {
+                    (this@NonBridgeLargeDataFragment.activity!! as FragmentNavigationManager).navigateTo(
+                            newInstance(
+                                    NonBridgeLargeDataArguments(true)
+                            ), true
```

Comment:
Put `true` on its own line:

```kotlin
                    (this@NonBridgeLargeDataFragment.activity!! as FragmentNavigationManager).navigateTo(
                            newInstance(
                                    NonBridgeLargeDataArguments(true)
                            ), 
                            true
```

Might help to use a named parameters in all these to make clear what the boolean are:

```kotlin
                    (this@NonBridgeLargeDataFragment.activity!! as FragmentNavigationManager).navigateTo(
                            newInstance(
                                    NonBridgeLargeDataArguments(infiniteBackstack = true)
                            ), 
                            addToBackstack  = true
```

Last thing : its technically safest to do the cast to `FragmentNavigationManager` in `onAttach` so that the cast fails as soon as possible if something is wrong:

```kotlin
private lateinit var fragmentNavigationManager

...

fun onAttach() {
    super.onAttach()
    fragmentNavigationManager = activity as FragmentNavigationManager
}
```

Reply:
Done, and ensuring an early fail makes sense.